library/ssl_misc.h

```diff
@@ -640,7 +640,10 @@ struct mbedtls_ssl_handshake_params
  * seem correct.
  */
 #if defined(MBEDTLS_ECDH_C) || defined(MBEDTLS_ECDSA_C)
+#if !defined(MBEDTLS_USE_PSA_CRYPTO) ||                         \
+    defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
```

Comment:
I'm surprised we need the ECDH context for EC J-PAKE. Do you know why? This should probably be explained in a comment here.

Reply:
It's a typo, bad copy paste from my first attempt around `ssl_parse_supported_point_formats_ext`, removing.